lib/between_meals/repo/git.rb

```diff
@@ -24,11 +24,8 @@ class Repo
     # Git provider
     class Git < BetweenMeals::Repo
       def setup
-        if File.exists?(File.expand_path(@repo_path))
-          @repo = Rugged::Repository.new(File.expand_path(@repo_path))
-        else
-          @repo = nil
-        end
+        fail unless File.exists?(File.expand_path(@repo_path + '.git'))
```

Comment:
you love to do this... this is the second time you've sent such a diff.

You cannot do this. We specifically do this for the case when we don't have a repo but need to create one, in GD.